### PR TITLE
Update metadata | Add .gitignore

### PR DESCRIPTION
**Pull request by vipulgupta2048**
**Updated following parameters**
`url`, `license`, `repository`, `tags`

Added .gitignore file.

### CODE REVIEW

**Comment by rhl-bthr** [comment]
Thanks

While you are traversing _each activity_, may I suggest implementing other things like:
 - fixing flake8 opportunities in the code
 - set pep 8 in the code
 - updating sugargame to v1.2 (if applicable) (possible only if you can run and test the activity)
 - adding/updating screenshots, if required (possible only if you can run the activity)
 - Build .xo files and report if there are any errors

**Comment by vipulgupta2048** [comment]
Thanks @Pro-Panda, I would love to contribute more as I traverse through the activities. Thanks for telling me.  

> fixing flake8 opportunities in the code

Easy enough, will try to fix them. I can. I use `pylint` would that be fine.

> set pep 8 in the code

Most activities I have seen follow it haphazardly, I can't promise. But I sure can take a look before pushing of the main files.

> updating sugargame to v1.2 (if applicable) (possible only if you can run and test the activity)

If you could provide me with some context about it, then it would be very helpful. Links will do. 

> adding/updating screenshots, if required (possible only if you can run the activity)

I am working on that, just installed sucrose. Would that be enough for testing?

> Build .xo files and report if there are any errors

Easy enough, consider it done. If I understand correctly, then run 
`python setup.py dist_xo` then `python setup.py build` Right?

**Comment by rhl-bthr** [comment]
TL;DR

PR reviewed, thanks

**Comment by quozl** [comment]
@vipulgupta2048, see also the sugar-docs repository for a checklist of activity maintenance, and also the projects link on github/sugarlabs.
